docs/changelog/2_x.rst

```diff
@@ -876,6 +876,76 @@ Other changes
 
 * UI: Fix loading indicators on tabs
 
+2.10
+====
+
+* Fix mismatch in session state after a ``ROLLBACK``
+
+* Fix ISE when doing set default on an abstract pointer
+  (:eql:gh:`#4843`)
+
+* Fix accesses to ``__type__`` from insert access policies
+  (:eql:gh:`#4865`)
+
+* Properly forbid aggregation in index expressions
+  (:eql:gh:`#4869`)
+
+* Fix `grouping` field when grouping by one key or nothing
+  (:eql:gh:`#4906`)
+
+* Fix two issues with mutation in free objects
+  (:eql:gh:`#4902`)
+
+* Only allow type names as the subject of an insert. (Previously dotted paths
+  were allowed, with nonsensical behavior.)
+  (:eql:gh:`#4922`)
+
+* Fix array arguments in HTTP interface
+  (:eql:gh:`#4956`)
+
+* Support multi properties in ``UNLESS CONFLICT ON``
+  (:eql:gh:`#4955`)
+
+* Fix polymorphic type tests on result of update
+  (:eql:gh:`#4954`)
+
+* Upgrade our python linters
```

Comment:
I would drop this and the `setuptools` note, as those are only relevant to EdgeDB devs.

Reply:
Oh yeah, oops